src/com/lazydroid/autoupdateapk/AutoUpdateApk.java

```diff
@@ -338,7 +340,12 @@ private static String getResponseText(InputStream inStream) {
 
 						in = new BufferedInputStream(conn.getInputStream());
 						String fname = result[1].substring(result[1].lastIndexOf('/')+1);
-						FileOutputStream out = context.openFileOutput( fname, Context.MODE_WORLD_READABLE);
+						FileOutputStream out = null;
```

Comment:
yes but I prefer keeping things explicit, given this is a workaround until Google fixes the issue!